skrifa/src/scale/scaler.rs

```diff
@@ -122,22 +123,30 @@ impl<'a> ScalerBuilder<'a> {
     pub fn build(mut self, font: &impl TableProvider<'a>) -> Scaler<'a> {
         self.resolve_variations(font);
         let coords = &self.context.coords[..];
-        let glyf = if let Ok(glyf) = glyf::Scaler::new(
+        let size = self.size.ppem().unwrap_or_default();
+        let outlines = if let Ok(glyf) = glyf::Scaler::new(
```

Comment:
I think this means that if I have cff *and* glyf with most of the glyphs in cff my font will perform exceedingly poorly. I'll grant you this is a weird font, but I think it should work. Maybe scaler should have a vector of Outlines in prioritized order; most of the time you would have only one but a particularly exciting font could have 0 or > 1.

Reply:
Oh hey or a COLR font that uses outlines from *both*! That's definitely a good idea right?

Reply:
The spec doesn't support this. Both `loca` and the charstrings index are defined to have `maxp.numGlyphs` entries and empty entries are valid (think any space glyph), so there is no way to specify sparse glyph sets.